rca/programmes/models.py

```diff
@@ -178,13 +169,12 @@ class ProgrammePage(BasePage):
             )
         ],
         blank=True,
-        null=True,
     )
 
-    facilities_intro = models.CharField(blank=True, null=True, max_length=150)
-    facilities_copy = models.TextField(blank=True, null=True, max_length=400)
+    facilities_intro = models.CharField(blank=True, max_length=150)
+    facilities_copy = models.TextField(blank=True, max_length=400)
     facilities_link = models.URLField(blank=True, null=True)
```

Comment:
`URLField` is based on a `CharField` so don't need `null=True` https://docs.djangoproject.com/en/2.2/ref/models/fields/#urlfield